db_schema.md

```diff
@@ -59,10 +61,11 @@
 - order (integer, used to sort prompts in link)
 - prompt (string, optional prompt above textbox/textarea)
 - required (boolean, if the javascript should force user to enter something)
-- text_area (boolean, true = use text area instead of single-row textbox)
 - var_name (string, name for captured user input, usable in actions)
 - link_id (integer, foreign key to links table)
 - link (auto reference to Link object that owns this prompt)
+- field_type (string, replaces text_area boolean for increased flexibility)
```

Comment:
My only concern with going with field_type, is that we haven't really thought through the other field types that we could support. As long as we stick with "input" and "textarea" I'm fine with this change.